Applications/DataExplorer/VtkVis/VtkVisPipelineView.cpp

```diff
@@ -85,26 +85,25 @@ void VtkVisPipelineView::contextMenuEvent( QContextMenuEvent* event )
         QMenu menu;
         QAction* addFilterAction = menu.addAction("Add filter...");
 
-        QAction* addLUTAction(nullptr);
-        QAction* addMeshingAction(nullptr);
         if (objectType == VTK_IMAGE_DATA)
         {
             // this exception is needed as image object are only displayed in the vis-pipeline
             isSourceItem = false;
-            addMeshingAction = menu.addAction("Convert Image to Mesh...");
+            QAction* addMeshingAction =
```

Comment:
Have you tested this? I suspect this won't work if the UI file doesn't find the QActions to build the menu?!  
(Some for all the other QAction-ptrs.)

Reply:
I compiled the DataExplorer and the testrunner. The testrunner worked fine. Is there another test for this?

Reply:
I guess this works, as the menu is populated during runtime (`menu.addAction()`).